legacy/bootstrap-cmake.sh

```
@@ -0,0 +1,64 @@
+
+cleanup() {
+  rm -rf "$1"
+}
+
+if [ -z "$1" ]
+then
+  echo "Usage: bootstrap-cmake.sh INSTALLDIR"
+  exit 1
+else
+  installdir=$(realpath "$1")
+  if [[ $? -ne 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)